README.md

```diff
@@ -1,42 +1,31 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+Mobile App VERSION
+Project Name
+MOBILE-VERSION
 
-# Project Name
+ Project Description.
 
->my-portfolio  
+Built With
+Major languages HTML and CSS
+Frameworks
+Stylelint e.t.c
 
+👤 Author1
+ADEBOWALE ADEGBOYE
+GitHub: @ademibowale
+Twitter: @ademibowale1
+LinkedIn: https://www.linkedin.com/in/adebowale-adegboye-143568221/
```

Comment:
- [ ] please try to fix your social media information by using the markdown language provided in the readme template 
example : `[put the name of the social media like : Github](put the URL of your profile here )` and it will be display as clickable link